.github/workflows/ci.yml

```diff
@@ -150,7 +150,9 @@ jobs:
             runs-on: macos-latest
 
           - backend: gtk
-            pre-command: "sudo apt-get update -y && sudo apt-get install -y python3-gi python3-gi-cairo gir1.2-gtk-3.0 python3-dev libgirepository1.0-dev libcairo2-dev pkg-config libfuse2"
+            pre-command: |
+              sudo apt update -y
+              sudo apt install -y libfuse2 pkg-config python3-dev libgirepository1.0-dev libcairo2-dev gir1.2-gtk-3.0
```

Comment:
I'm pretty sure libfuse2 isn't needed any more - it was only required because we were using AppImages.